components/nimbus/src/updating.rs

```diff
@@ -38,41 +38,3 @@ pub fn read_and_remove_pending_experiments(
     // None is "there are no pending updates".
     Ok(experiments)
 }
-
-// This test crashes lmdb for reasons that make no sense, so only run it
-// in the "safe mode" backend.
-#[cfg(feature = "rkv-safe-mode")]
```

Comment:
heh, this wasn't under a `cfg[test]` before 😆 Now it won't be compiled with non-test code 🎉 🎉